quickstarts/calico/config.yml

```
@@ -1,17 +1,17 @@
 id: a4383e84-ad92-4771-8f58-d0a459542be0
-slug: calico-newrelic
-title: Calico-Newrelic
+slug: calico-prometheus
```

Let's keep with as `calico-newrelic` otherwise this will change the URL that the quickstart is available at.
```suggestion
slug: calico-newrelic
```